.github/workflows/checks.yml

```diff
@@ -64,7 +64,7 @@ jobs:
 
       - name: Compile all code with fatal warnings for Java 11, Scala 2.12 and Scala 2.13
         # Run locally with: sbt 'clean ; +Test/compile ; +It/compile'
-        run: sbt "; Test/compile; It/compile; headerCheckAll"
```

Comment:
I added the headerCheckAll but it was unnecessary - verifyCodeStyle is called earlier and I've now made that use headerCheckAll